Description: Upstream is providing it's own clang_includes directory.
 This patch is telling ycmd to use system clang include directory.

--- a/ycmd/completers/cpp/flags.py
+++ b/ycmd/completers/cpp/flags.py
@@ -284,8 +284,11 @@
 
 
 def _SpecialClangIncludes():
-  libclang_dir = os.path.dirname( ycm_core.__file__ )
-  path_to_includes = os.path.join( libclang_dir, 'clang_includes' )
-  return [ '-isystem', path_to_includes ]
+  import glob
+  import re
+  for clang_include_dir in glob.glob('/usr/lib/clang/*'):
+    if re.match('3\.(6|7|8)', os.path.basename(clang_include_dir)):
+      return [ '-isystem', clang_include_dir + '/include' ]
+  return []
 
 
